aten/src/ATen/native/Unique.cpp

```diff
@@ -0,0 +1,45 @@
+// Returns unique elements of input tensor.
+
+#include <unordered_map>
+#include <unordered_set>
+
+#include "ATen/ATen.h"
+
+namespace at {
+namespace native{
+
+std::tuple<Tensor, Tensor> unique(
+    const Tensor& self, const bool sorted, const bool return_inverse) {
+  std::unordered_set<int64_t> set(
+      self.data<int64_t>(), self.data<int64_t>() + self.numel());
+  Tensor output = self.type().tensor({static_cast<long long>(set.size())});
+  
+  if (!sorted) {
+    std::copy(set.begin(), set.end(), output.data<int64_t>());
+  } else {
+    std::vector<int64_t> vec(set.begin(), set.end());
+    std::sort(vec.begin(), vec.end());
+    std::copy(vec.begin(), vec.end(), output.data<int64_t>());
+  }
+
+  Tensor inverse_indices;
+  if (!return_inverse) {
+    inverse_indices = self.type().toScalarType(kLong).tensor({0});
+    
```

Comment:
Nit: remove newline Actually we have a `.clang-format` here, so you can just auto-format it (e.g. using Nuclide)

Reply:
Also flip the cases again

Reply:
Done